fili-core/src/main/java/com/yahoo/bard/webservice/data/config/metric/makers/AggregationAverageMaker.java

```diff
@@ -67,36 +74,51 @@
      * @param innerGrain  The time grain across which queries should aggregate.
      */
     public AggregationAverageMaker(MetricDictionary metrics, ZonelessTimeGrain innerGrain) {
-        super(metrics);
+        this(metrics, innerGrain, BuiltInMetricProtocols.getStandardProtocolSupport());
+    }
+
+    /**
+     * Constructor.
+     *
+     * Any aggregation average maker will disable the protocol Aggregation.
```

Comment:
I think we intend `Reaggregation` here not `Aggregation`. Also, we may want to include a link to where the
protocol is defined in BuiltInMetricProtocols.